src/main/java/de/leonhard/storage/util/ClassWrapper.java

```diff
@@ -82,7 +82,7 @@
   }
 
   @UtilityClass
-  public class LONG {
+  public static class LONG {
```

Comment:
Doesn't the Utility Class do the `static` keyword for you? Or is that only on every method and field inside the class?

Reply:
Yes thats right, the static is useless here.

Reply:
It doesn't make the class static, but it does make it final:
https://projectlombok.org/features/experimental/UtilityClass

Reply:
Yes, just read `If the class is an inner class, the class is also marked static.` on the docs. I'll remove them.